lib/aes/aes.rb

```diff
@@ -94,8 +94,7 @@ def _random_seed(size=32)
         if defined? OpenSSL::Random
           return OpenSSL::Random.random_bytes(size)
         else
-          chars = ("a".."z").to_a + ("A".."Z").to_a + ("0".."9").to_a
-          (1..size).collect{|a| chars[rand(chars.size)] }.join        
+          (1..size).collect{ rand(256).chr }.join
```

Comment:
When using a modulo based upon a-z,A-Z,0-9 which comprises 62 characters, this effectively yields log2(62) (about 5.95) bits per character of randomness. rand(256) will yield a full 8 bits per character, inline with what OpenSSL::Random.random_bytes would return.